webapp/src/features/UserItem/ActivityHistory.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+import { Row, Col, Button, Image } from 'react-bootstrap'
+import Icons from '../../assets/icons'
+import Table from '../../widgets/Table'
+import classNames from 'classnames'
+import SortingCustomIcon from '../../widgets/SortingCustomIcon'
+
+const ActivityHistory = () => {
+	const eventIconFormatter = (row) => {
+		let iconName = ''
+		let iconClass = ''
+		switch (true) {
+			case row.includes('Credit'):
+				iconName = 'orangeDollarIcon'
+				iconClass = 'bg-warning'
+				break
+			case row.includes('Refunded'):
+				iconName = 'blueDollarIcon'
+				iconClass = 'bg-secondary'
+				break
+			case row.includes('Downgraded'):
+				iconName = 'redArrowDownIcon'
+				iconClass = 'bg-danger'
+				break
+			case row.includes('Paid'):
+			case row.includes('Subscription Creation'):
+				iconName = 'greenDollarIcon'
+				iconClass = 'bg-success'
+				break
+			case row.includes('Churn'):
+				iconName = 'redDollarIcon'
+				iconClass = 'bg-danger'
+				break
+			case row.includes('Upgraded'):
+				iconName = 'greenArrowUpIcon'
+				iconClass = 'bg-success'
+				break
+			default:
+				break
+		}
+		return (
+			<div className='d-flex align-items-center'>
+				<div className='arrow-bubble bubble-sm position-relative rounded-circle overflow-hidden d-flex align-items-center justify-content-center mr-2'>
+					<Image src={Icons[iconName]} alt='events-icon' />
+					<span
+						className={classNames(iconClass, 'position-absolute h-100 w-100')}
```

Comment:
No need of classNames.

Reply:
classNames need to be provided cause iconClass is the className which changes according to the data....I have tried using `` formatting but it does not support in className.